.babelrc

```diff
@@ -15,7 +15,10 @@
   ],
   "env": {
     "test": {
-      "presets": ["@babel/preset-env"]
+      "presets": ["@babel/preset-env"],
+      "plugins": [
+        ["babel-plugin-styled-components", { "ssr": false, "displayName": false }]
+      ]
```

Comment:
Had to add this because of an issue with jest-styled-components. See here: https://github.com/styled-components/jest-styled-components/issues/294